Source/DafnyLanguageServer.Test/Util/ClientBasedLanguageServerTest.cs

```diff
@@ -155,11 +157,14 @@ protected ClientBasedLanguageServerTest(ITestOutputHelper output, LogLevel dafny
     await client.WaitForNotificationCompletionAsync(documentItem.Uri, cancellationToken);
     var compilation = (await Projects.GetLastDocumentAsync(documentItem))!;
     Assert.NotNull(compilation);
-    var expectedDiagnostics = compilation.GetDiagnostics(documentItem.Uri.ToUri()).Select(d => d.ToLspDiagnostic()).ToList();
+    var expectedDiagnostics = compilation.GetDiagnostics(documentItem.Uri.ToUri()).
+      Select(d => d.ToLspDiagnostic()).
+      OrderBy(d => d.Range.Start).ToList();
```

Comment:
Minor style preference, you do what you want here, but I prefer the dot to be put before the method name rather than after the object, when inserting newlines. I know it makes it looks like sentences, but that would be true only if the last call finished with a dot.

Reply:
I don't have a strong opinion on this, but I would rather discuss this outside of this PR and then try to have a formatting rule for.